lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -255,11 +255,20 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
 
         if (isset($xmlRoot->embedded)) {
             foreach ($xmlRoot->embedded as $embeddedMapping) {
+                $columnPrefix = isset($embeddedMapping['column-prefix'])
+                    ? (string) $embeddedMapping['column-prefix']
+                    : null;
+
+                $preventPrefixing = (
```

Comment:
Wrong. It will return true for a valid prefix. I only need ensure that false or 0 was given.